memo_app.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'debug'
+require 'sinatra'
+require 'erb'
+require 'securerandom'
+
+helpers do
+  def h(text)
+    ERB::Util.h(text)
+  end
+end
+
+get '/memo' do
+  @memos = read_memos
+  erb :memo_list
+end
+
+get '/memo/new' do
+  erb :memo_new
+end
+
+post '/memo/new' do
```

Comment:
```suggestion
post '/memo' do
```

Railsとかだと `/new` は付けないので、強いこだわりがないならあわせた方がいいかもしれません

Reply:
なるほど…
`/new`を付けないように修正しました！